docs/pages/management/operations/ca-rotation.mdx

```diff
@@ -240,47 +240,16 @@ metadata file and make it available to your service provider.
 
 ### `oidc_idp`
 
-The `oidc_idp` CA signs messages sent by the Teleport OIDC IdP integration. AWS
-verifies these messages to authenticate your Teleport account for features like
-EICE EC2 Access, External Audit Storage, Auto-Discovery, and AWS Sync for Access
-Graph. There are two ways to serve the public key for the Teleport `oidc_idp`
-certificate authority:
+The `oidc_idp` CA signs messages sent by the Teleport OIDC IdP integration.
+Relying parties (e.g., AWS) verify these messages to authenticate your Teleport
+account for features like External Audit Storage, Auto-Discovery, and AWS Sync
+for Access Graph. 
 
-- From the `/.well-known/jwks-oidc` path of the Teleport Proxy Service Web API.
-- From an Amazon S3 bucket.
+The Teleport Proxy Service serves the public key for the Teleport `oidc_idp`
+certificate authority from the `/.well-known/jwks-oidc` path of the Web API.
```

Comment:
nit: I would mention `/.well-known/openid-configuration` as well, which includes the path to the jwks uri, e.g. mine looks like:
```json
{
  "issuer": "https://alpha.devteleport.com",
  "jwks_uri": "https://alpha.devteleport.com/.well-known/jwks-oidc",
  "claims": [
    "iss",
    "sub",
    "obo",
    "aud",
    "jti",
    "iat",
    "exp",
    "nbf"
  ],
  "id_token_signing_alg_values_supported": [
    "RS256"
  ],
  "response_types_supported": [
    "id_token"
  ],
  "scopes_supported": [
    "openid"
  ],
  "subject_types_supported": [
    "public",
    "pair-wise"
  ]
}
```